migrations/20190116170623_generate_students_table.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('students', table => {
+        table.increments();
+        table.string('name').notNullable();
+        table
+            .integer('cohort_id')
+            .unsigned()
+            .references('id')
+            .inTable('cohorts');
```

Comment:
You're alternatively able to use the slightly shorter `on` method here:
```javascript
  // ...
  .references('id')
  .on('cohorts')
```
or skip the extra method altogether:
```javascript
  // ...
    .references('cohorts.id')
```